desktop/release-notes.md

```diff
@@ -0,0 +1,782 @@
+---
+description: Release notes for Docker Desktop for Mac, Linux, Windows
+keywords: Docker desktop, release notes, linux, mac, windows
+title: Docker Desktop release notes
+toc_max: 2
+redirect_from:
+- /docker-for-mac/release-notes/
+- /docker-for-mac/edge-release-notes/
+- /desktop/mac/release-notes/
+- /docker-for-windows/edge-release-notes/
+- /docker-for-windows/release-notes/
+- /desktop/windows/release-notes/
+- /desktop/linux/release-notes/
+---
+
+This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
+
+> **Note**
+>
+> The information below is applicable to all platforms, unless stated otherwise.
+
+Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
+
+## Docker Desktop 4.8.2
+2022-05-17
+
+> Download Docker Desktop 
+>
+> [For
+> Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){:
+> .button .primary-btn }
+> [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
+> [Mac with Apple
+> chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
+> .button .primary-btn }
+> [DEB](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-amd64.deb?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
+> .button .primary-btn }
+> [RPM](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.rpm?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
+> .button .primary-btn }
+> [Arch package](https://desktop.docker.com/linux/main/amd64/docker-desktop-4.8.2-x86_64.pkg.tar.zst?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-linux-amd64){:
+> .button .primary-btn }
```

Comment:
Do we support arch at the moment? The Linux installation page only lists DEB and RPM: https://docs.docker.com/desktop/linux/install/
@p1-0tr @aiordache?

Reply:
On that same page a little further down it mentions a DD package for Arch based distributions but highlights it's experimental

Reply:
@christophermclellan any thought?

Reply:
We could change to 'Arch (experimental)' here ?